components/price/price.scss

```diff
@@ -0,0 +1,55 @@
+@import "../../themes/theme";
+
+.ui-price {
+  color: $tx-generic-color-primary-dark;
+  font-family: $tx-generic-font-primary-font-family, $tx-generic-font-primary-generic-family;
+  display: inline-block;
+}
+
+.ui-price__value-delimiter {
+  align-items: flex-end;
+  display: flex;
+  flex-flow: row nowrap;
+  flex: 0 0 auto;
+}
+
+.ui-price__currency {
+  display: flex;
+  font-size: $tx-price-font-size-currency;
+  flex: 0 0 auto;
+
+  &.ui-price__currency-left {
```

Comment:
Well it's both a modifier and an element's value... But I'd go for modifier...

Should be separated by `_`, is that?